layout/pulse/layer-menu-native/index.js

```diff
@@ -0,0 +1,24 @@
+import { Component, createElement } from 'react';
+import { connect } from 'react-redux';
+
+import * as actions from './actions';
+import * as reducers from './reducer';
+import initialState from './initial-state';
+
+import LayerMenuNativeComponent from './component';
+
+const mapStateToProps = state => ({
+  layerActive: state.pulse.layerActive
+});
+
+class LayerMenuNativeContainer extends Component {
```

Comment:
This is unnecessary, you can pass directly the component to the connect. In fact, you are using the same name...eslint should be saying something